docs/yaml/elementary/int.yml

```diff
@@ -14,3 +14,12 @@ methods:
 - name: to_string
   returns: str
   description: Returns the value of the number as a string.
+
+  optargs:
+    fill:
+      type: int
+      description: |
+        Left fill the string with zeros until it reaches the length 
+        specified by this argument. The string is left untouched if the
+        value provided is less than the original length of the string.
```

Comment:
Details for negative number is more useful here than in release notes, IMHO.

Reply:
Updated